### PR TITLE
Add Romanian translation

### PR DESCRIPTION
**Pull request by egyptianbman**
@awreeoh on Discord was kind enough to provide us with Romanian
translations!

### CODE REVIEW

**Review by cryptorex — APPROVED** [review]
that romanian is very accurate

File: translation/languages/ro.csv [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+BitcoinZ: The true Bitcoin 2.0,BitcoinZ: Adevăratul Bitcoin 2.0
+Welcome to the decentralization,Bine ai venit la descentralizare. 
+Start Mining,Incepe sa minezi
+Latest News,Ultimele ştiri
+Donate,Donează
+About the <span>coin</span>,Despre <span>monedă</span>
+Important Details,Detalii importante
+- Backup your wallet in many locations<br>- Store big amount of coins in cold wallets in anonymous addresses<br>- Don't give / send anyone your private keys. Private keys needs to be secured. You can recover your wallets. If you give or send anyone your private key please consider it as a red flag and quickly move your coins to other address,"- Fă-ţi un back-up la portofel in mai multe locaţii.<br>
+- Depozitează sume mari in portofele offline în adrese anonime.<br>
+- Niciodată nu primi / trimite nimănui cheile tale private. Cheile private trebuiesc ţinute în siguranţă. Îşi poţi recupera portofelele cu ajutorul lor. Dacă ii dai sau ii trimiâi cuiva cheia ta privata, te rugăm consideră acest lucru un semnal de alarmă si mută-ţi cât mai repde monedele în altă adresă. "
```
Comment (faneaatiku) [inline translation/languages/ro.csv]: "trimiâi" should be "trimiți"

File: translation/languages/ro.csv [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+BitcoinZ is <span>Bitcoin 2.0</span>,BitcoinZ este <span>Bitcoin 2.0</span>
+"- Bitcoin technology<br>- Anonymous transactions (zk-SNARKs privacy)<br>- Bitcoin has all transactions publicly available.<br>- decentralized GPU mining (ASIC miners are banned, hardforks to prevent ASIC miners are allowed)<br>- Decentralized development / All contributors are volunteers<br>- Big blocks -> blocks are similar to BCC / BCH","- Tehnologie Bitcoin<br>
+- Tranzacţii anonime (anonimitate oferită de zk-SNARKs)<br>
+- Bitcoin afişează toate tranzacţiile pe blockchain. <br> 
+- Minare desecentralizată cu ajutorul GPU-ului (echipamentele ASIC sunt banate, hardfork-uri permise doar pentru a preveni ASIC-urile) <br>
+- Dezvoltare descentralizată / Toţi cei care contribuie la proiect sunt voluntari <br> 
+- Block-uri mari -> block-uri similare cu cele de BCC / BCH"
+Active <span>Community</span>,<span>Comunitate</span> activă
+"Our community is active and we are always developing. <br>You can follow us in social networks, on the forum, as well as Slack","Avem o comunitate activă care dezvolta continuu. <br>
```
Comment (faneaatiku) [inline translation/languages/ro.csv]: "dezvolta" to "dezvoltă"

File: translation/languages/ro.csv [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+Ne poţi urmări in reţelele sociale, pe forum sau pe Slack. "
```
Comment (faneaatiku) [inline translation/languages/ro.csv]: "in" to "în"

File: translation/languages/ro.csv [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+Public Relations,Relaţii publice
+why choose <span>BitcoinZ</span>,De ce să alegi <span>BitcoinZ</span>
+Here are a few <span>reasons</span>,Iată nişte <span>motive</span>
+WARNING: Always remember about the pitfalls and keep your backup keys in a safe place!,ATENŢIE: Ţine cont de capcane şi ţine-ţi cheile de back-up într-un loc sigur!
+Decentralized <span>Mining</span>,<span>Minat<span> descentralizat
+As a miner you should not use the biggest pools to follow main principles.<br>Easy to mine<br>Equihash algorithm.<br>Decentralized Exchanges,Ca miner nu trebuie să foloseşti cele mai mari pool-uri pentru a urmări principiile. <br>Uşor de minat<br>Algoritm Equihash<br> Exchange-uri descentralizate
+No <span>Pre-Mine</span>,Fără <span>pre-minare</span>
+Pure coin. No pre-mine.,Monedă pură. Fără pre-minare
+21 billion coins! Everyone in the world can have at least one BTCZ!,21 de miliarde de monede! Toată lumea poate deţine măcar 1 BTCZ!
+Our community is actively developing and growing every day.,Comunitatea noastră dezvoltă şi creşte în fiecare zi.
```
Comment (faneaatiku) [inline translation/languages/ro.csv]: there should be a "se" between "noastră" and "dezvoltă" and become "noastră se dezvoltă"

**Review by faneaatiku — CHANGES_REQUESTED** [review]

**Comment by faneaatiku** [comment]
It looks good leaving aside what i suggested.

**Comment by awreeoh** [comment]
sorry about those mistakes guys... it was also a hard day at work :")

**Comment by egyptianbman** [comment]
@faneaatiku let me know if this looks good now; thanks!

**Comment by faneaatiku** [comment]
Everything fine! Thanks Bam, thanks fellow awreeoh!

**Pull request merged**